.vib/apache/goss/goss.yaml

```
@@ -10,3 +10,4 @@ gossfile:
   ../../common/goss/templates/check-files.yaml: {}
   ../../common/goss/templates/check-linked-libraries.yaml: {}
   ../../common/goss/templates/check-sed-in-place.yaml: {}
+  ../../common/goss/templates/check-sdpx.yaml: {}
```

This is actually `spdx`
```suggestion
  ../../common/goss/templates/check-sdpx.yaml: {}
```

---

Thanks! Typo fixed